src/docs.md

```diff
@@ -205,6 +205,19 @@ const unsubscribe = cadenzaClient.on('editGeometry:ok', (event) => ...);
 unsubscribe();
 ```
 
+### Subscribe to map extent change
+
+Subscribe to `extent:change` event to be notified about map extent changes. The event details contain the new map extent (transformed to EPSG:4326):
```

Comment:
It's `change:extent`. But if I look at the example - maybe it's not really necessary, because the event is documented in the api docs and we already have an example for subscribing to an event in TS. What do you think?

If you keep the example (and resolved the above comment), you should use the `CadenzaChangeExtentEvent` type.